README.md

```diff
@@ -146,9 +146,31 @@ wraps an item using this class, and proceeds to handle it with the provided inte
 interface, providing a `dict`-like API to manipulate data for the object it wraps
 (which is modified in-place).
 
-Some additional methods are available:
+**Attributes**
 
-`get_field_meta(field_name: str) -> MappingProxyType`
+* _class attribute_ `itemadapter.adapter.ItemAdapter.ADAPTER_CLASSES`
```

Comment:
In line with methods below:

```suggestion
* _class attribute_ `ADAPTER_CLASSES`
```

Maybe it should be type-annotated as well?

Reply:
I annotated it at first, but felt like `class attribute collections.deque itemadapter.adapter.ItemAdapter.ADAPTER_CLASSES` was kind of a mouthful. Your suggestion helps to keep it short, thanks.